.pre-commit-config.yaml

```diff
@@ -37,7 +37,7 @@ repos:
     rev: v1.7.1
     hooks:
     -   id: mypy
-        additional_dependencies: [types-toml]
+        additional_dependencies: [types-toml, pydantic]
```

Comment:
**suggestion:** have mypy read its configuration in the pyproject.toml

try this instead
```suggestion
        additional_dependencies: [
            types-toml,
            tomli, # to read config from pyproject.toml
        ]
```

Reply:
To be continued on #25